src/app/utils/CacheModule.ts

```diff
@@ -28,7 +28,7 @@ interface CachedData {
 }
 
 export class CacheModule {
-	private staleTime = 5 * 60 * 1000 // 6 Minutes
+	private staleTime = 5 * 60 * 1000 // 5 Minutes
```

Comment:
While this is set to 5 minutes as a default, the constructor uses 6 minutes as default for the `staleTime` parameter.
I did not want to change any functionality within this commit so I have just updated the comment here.

If you want to harmonize both values, let me know.

Reply:
You can remove the initialization of the variable here, you are right we set it in the constructor. No need to define it here